site/source/user-manual/connecting/connecting-lan/lan-os/lan-android.rst

```diff
@@ -4,10 +4,24 @@
 Trusting Embassy CA on Android
 ==============================
 
-.. warning:: This is only possible on Android 12+, which is not yet available on Graphene/Calyx.
+.. note:: This will work on stock Android, and equally well on CalyxOS, GrapheneOS or LineageOS.
 
-#. On your Android device, go to *Settings > Security > Advanced > Encryption and Credentials > Install from Storage* and select your "Embassy Local Root CA" certificate.
+.. warning:: This is only possible on Android version 12 or greater.  For LineageOS, that corresponds to v19+.
 
-    .. figure:: /_static/images/ssl/android/droidLAN0.png
-        :width: 30%
-        :alt: Install certificate
+.. tabs::
+
+    .. group-tab:: Android v12
```

Comment:
On next update, we ought to have the latest version as the default tab, and probably in general